install.sh

```diff
@@ -1,42 +1,96 @@
 #!/usr/bin/env bash
 
-read -p "Do you want me to install Group Butler Bot? (Y/N): "
+# Lua Version
+LUA=5.2
 
-if [ "$REPLY" != "Y" ]; then
-	echo "Exiting..."
-else
-	echo -e "\e[1;36mUpdating packages\e[0m"
-	sudo apt-get update -y
+# Dependencies and libraries
+NATIVE="libreadline-dev libssl-dev lua$LUA luarocks liblua$LUA-dev git make unzip redis-server curl libcurl4-gnutls-dev deborphan"
+ROCKS="luasocket luasec redis-lua lua-term serpent cjson Lua-cURL"
 
-	echo -e "\e[1;36mInstalling dependencies\e[0m"
-	sudo apt-get install libreadline-dev libssl-dev lua5.2 luarocks liblua5.2-dev git make unzip redis-server curl libcurl4-gnutls-dev -y
+# Color variables
+Red='\033[0;31m'
+Green='\033[0;32m'
+Orange='\033[0;33m'
+Blue='\033[0;34m'
+Purple='\033[0;35m'
+Cyan='\033[0;36m'
+BRed='\033[1;31m'
+BGreen='\033[1;32m'
+BOrange='\033[1;33m'
+BBlue='\033[1;34m'
+BPurple='\033[1;35m'
+BCyan='\033[1;36m'
+Default='\033[0m'
 
-	echo -e "\e[1;36mInstalling LuaRocks from sources\e[0m"
+read -p "Do you want me to install Group Butler Bot? (Y/N): "
 
-	git clone http://github.com/keplerproject/luarocks
-	cd luarocks
-	./configure --lua-version=5.2
-	make build
-	sudo make install
-	cd ..
+case $REPLY in [yY])
+	# Update System
+	clear
+	echo -en "${Cyan}Do you want to upgrade your system (Y/N): ${Default}"
+	read REPLY
+	if [[ $REPLY == [yY] ]]; then
+		sudo apt-get update
+		sudo apt-get upgrade -y
+		sudo apt-get dist-upgrade -y
```

Comment:
uh please don't do that. This fails on any non-debian based distros and is potentially dangerous.